src/LibraryViewExtension/web/library/library.html

```diff
@@ -55,6 +55,23 @@
         //Update the view with contents
         refreshLibraryView(libController);
 
+        libController.searchLibraryItemsHandler = function (text, callback) {
+            let url = csharpController.searchLibraryItems(text);
```

Comment:
We can directly have a url to provide a search service as `http://localhost/nodesearch/{searchText}` and register a provider for `http://localhost/nodesearch/`